File: 00_이원영/1152.py
string = input()

print(len(string.split()))

'''
# split을 서술해서 아래와 같이 구현해보려했는데 계속 틀렸다고 뜸..ㅠ

bl = 0
a = ' '
b = ' '
c = ' '
str = input()

for i in str :
    a = b
    b = c
    c = i
    if b == ' ' and a != ' ' and c != ' '  :
        bl += 1

print(bl+1)

'''

File: 18_정지윤/1152.java
import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;
public class Main {
    private void solve() {
        try (BufferedReader br = new BufferedReader(new InputStreamReader(System.in))) {
            String[] str = br.readLine().split(" ");
            if (str.length <= 0) {
                System.out.println(0);
            } else {
                if (str[0].equals("")) {
                    System.out.println(str.length-1);
                } else {
                    System.out.println(str.length);
                }
            }
        }catch (IOException e) {
            e.printStackTrace();
        }
    }
    public static void main(String[] args){
        new Main().solve();
    }
}